catalog/core/catalog-core-api/src/main/java/ddf/catalog/event/Subscription.java

```diff
@@ -36,4 +36,18 @@
      */
     public DeliveryMethod getDeliveryMethod();
 
+    /**
```

Comment:
I still don't love this approach, as we're now providing implementation hints/guidance from the interface; however, it's probably okay.